### PR TITLE
[uss_qualifier] Replace has_private_address by new resource TestExclusionResource

### PR DESCRIPTION
**Pull request by mickmis**
Address #739 

Notes:
- Many files are changed but most are configurations and suites propagating the new resource into the scenarios that use it
- For NetRID DSS resources the field `has_private_address` is kept but only because it is used to determine the value of `local_debug`. A soon-to-follow PR will move `local_debug` into `TestExclusionsResource` to clean that up completely.
- Behavior change: the 'publicly adressable' check is now skipped, before it succeeded when the flag was set.
- For the NetRID DSS interoperability scenarios, the severity of the 'publicly adressable' check is now set through the .md file and is now high (before: medium).

### CODE REVIEW

**Comment by mickmis** [comment]
Switching to draft until CI errors are resolved.

File: monitoring/uss_qualifier/configurations/interuss/library/environment.yaml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # The resources in this file describe the system/environment under test and should not change the test being run.
 # This file defines the environment use to verify releases on a deployment of two DSSs pooled in two different clouds.
 
+# ===== Test Exclusions =====
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/configurations/interuss/library/environment.yaml:4]: We have a test exclusions definition in library/resources.yaml and I think that's where it should go.  The references I see all $ref to the definition in library/resources.yaml.  Test exclusions generally shouldn't be part of the test environment (the main motivation for this change).  I think this block can simply be deleted?

File: monitoring/uss_qualifier/scenarios/astm/netrid/v19/dss_interoperability.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ A resources.astm.f3411.DSSInstanceResource containing the "primary" DSS instance
 
 A resources.astm.f3411.DSSInstancesResource containing at least two DSS instances complying with ASTM F3411-19.
 
+### test_exclusions
+
+A [resources.dev.TestExclusionsResource](../../../../resources/dev/test_exclusions.py) containing test exclusions parameters like whether private addresses are allowed.
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/scenarios/astm/netrid/v19/dss_interoperability.md:24]: Let's note that this resource is optional (same in other scenarios as well)

**Review by BenjaminPelletier — APPROVED** [review]
Thanks for keeping the scope down on this PR -- even though it does affect a lot of files, it was super easy to review because it only does one core thing (plus a small behavior fix).  Two small comments, but then feel free to merge without any additional approval after you think those are addressed or inapplicable.

File: monitoring/uss_qualifier/configurations/interuss/library/environment.yaml (hunk 1 above)
Comment (mickmis) [reply monitoring/uss_qualifier/configurations/interuss/library/environment.yaml:4]: Oh yes I missed that. Will remove it.

File: monitoring/uss_qualifier/scenarios/astm/netrid/v19/dss_interoperability.md (hunk 2 above)
Comment (mickmis) [reply monitoring/uss_qualifier/scenarios/astm/netrid/v19/dss_interoperability.md:24]: Added in the 3 scenarios.

**Pull request merged**